website/static/js/nodeControl.js

```diff
@@ -213,7 +213,7 @@ var ProjectViewModel = function(data) {
             title: 'Create identifiers',
             message: '<p class="overflow">' +
                 'Are you sure you want to create a DOI and ARK for this ' +
-                self.nodeType + '?',
+                $osf.htmlEscape(self.nodeType) + '?',
```

Comment:
Yeah... for this and addon names it could have gone either way. 

Realistically, we should just never construct HTML outside the front end (these exceptions to a uniform rule are what makes the display layer so dicey). 

But since we have a lot of copy/pasting, I figured I'd try to be consistent with the examples we had, even aside from the variables that really did need it. :)